src/tests.zig

```diff
@@ -143,17 +143,36 @@ test "serializes a structure without variable fields" {
     try expect(std.mem.eql(u8, list.items, serialized_data[0..]));
 }
 
-test "(de)serializes a structure with variable fields" {
+test "(de)serializes a runtime structure with variable-size fields" {
     // Taken from ssz.cr
     const Person = struct {
-        name: []const u8,
+        name: []u8,
         age: u8,
```

Comment:
interestingly, this also causes the same "comptime ref" issue, and so I need a more complete workaround for it.